data/examples/example-1.xml

```diff
@@ -0,0 +1,45 @@
+<Bundle xmlns="http://hl7.org/fhir" xmlns:h="http://www.w3.org/1999/xhtml"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://hl7.org/fhir http://hl7.org/fhir/R4/fhir-single.xsd http://www.w3.org/1999/xhtml http://hl7.org/fhir/R4/fhir-xhtml.xsd">
+	<id value="bundle-example-1" />
+	<type value="transaction" />
+
+	<!-- A Person entry to represent our Patient/patient as a person. Linked to Patient/patient. -->
+	<!-- https://www.hl7.org/fhir/person.html -->
+	<entry>
+		<resource>
+			<Person>
+				<id value="person" />
+				<link>
+					<target>
+						<reference value="Patient/patient" />
+					</target>
+				</link>
+			</Person>
```

Comment:
Test comment, and how it will look when resolved. let's give our patient a name.